lib/shared.ts

```diff
@@ -61,6 +61,8 @@ export interface User {
   webauthn_registrations: WebAuthnRegistration[];
   totps: TOTP[];
   crypto_wallets: CryptoWallet[];
+  trusted_metadata?: Record<string, any>; // eslint-disable-line @typescript-eslint/no-explicit-any
+  untrusted_metadata?: Record<string, any>; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
[sand] Discussed synchronously: To make sure these don't desync across files (and to keep the ESLint directive in one spot), consider defining a `UserMetadata` type alias somewhere.